### ISSUE TITLE
pVACfuse top score filter: `KeyError: 'Fusion Position'`

### ISSUE THREAD

**Issue by vladsaveliev** [griffithlab/pVACtools#194]
* pvactools version: master
* Python version: 3.6
* Operating System: linux

**Describe the bug**
When running `pvacfuse run` with `-t` option, it fails at this line: https://github.com/griffithlab/pVACtools/blob/935dbb31a92377dba5bc580db962dbfecc2be140/lib/top_score_filter.py#L23 
Throwing a `KeyError` as the `Fusion Position` column in missing. When running without `-t`, it finishes without errors, but indeed the column is missing. Here are the columns I see in `MHC_Class_I/mysample.final.tsv` (also attached this file in an archive):

```
Chromosome      Start   Stop    Reference       Variant Transcript      Ensembl Gene ID Variant Type    Mutation        Protein Position        Gene Name       HLA Allele      Peptide Length  Sub-peptide Position    Mutation Position    MT Epitope Seq   WT Epitope Seq  Best MT Score Method    Best MT Score   Corresponding WT Score  Corresponding Fold Change       Tumor DNA Depth Tumor DNA VAF   Tumor RNA Depth Tumor RNA VAF   Normal Depth    Normal VAF      Gene Expression       Transcript Expression   Median MT Score Median WT Score Median Fold Change      NetMHCcons WT Score     NetMHCcons MT Score     SMM WT Score    SMM MT Score    SMMPMBEC WT Score       SMMPMBEC MT Score
```

I'm wondering that happens because of this line here: https://github.com/griffithlab/pVACtools/blob/935dbb31a92377dba5bc580db962dbfecc2be140/lib/input_file_converter.py#L406

So fusion position is in fact populated into the `Protein Position` column instead?

**To Reproduce**

Input `mysample.anno.subset.bedpe` is attached in an archive.

```
pvacfuse run mysample.anno.subset.bedpe mysample "HLA-C*05:01" SMM pvacfuse_results -t -e 9 --iedb-install-directory /my/path/to/iedb -k
```

**Log Output**
```
Executing MHC Class I predictions
Converting .bedpe to TSV
Completed
Splitting TSV into smaller chunks
Splitting TSV into smaller chunks - Entries 1-1
Completed
Generating Variant Peptide FASTA and Key Files
Generating Variant Peptide FASTA and Key Files - Entries 1-2
Completed
Processing entries for Allele HLA-C*05:01 and Epitope Length 9 - Entries 1-2
Running IEDB on Allele HLA-C*05:01 and Epitope Length 9 with Method SMM - Entries 1-2
Completed
Parsing IEDB Output for Allele HLA-C*05:01 and Epitope Length 9 - Entries 1-2
Completed
Combining Parsed IEDB Output Files
Completed
Running Binding Filters
Completed
Running Top Score Filter
Traceback (most recent call last):
  File "/g/data3/gx8/extras/vlad/miniconda/envs/pvac/bin/pvacfuse", line 11, in <module>
    sys.exit(main())
  File "/g/data3/gx8/extras/vlad/miniconda/envs/pvac/lib/python3.6/site-packages/tools/pvacfuse/main.py", line 49, in main
    args[0].func.main(args[1])
  File "/g/data3/gx8/extras/vlad/miniconda/envs/pvac/lib/python3.6/site-packages/tools/pvacfuse/run.py", line 99, in main
    pipeline.execute()
  File "/g/data3/gx8/extras/vlad/miniconda/envs/pvac/lib/python3.6/site-packages/lib/pipeline.py", line 364, in execute
    self.top_result_filter()
  File "/g/data3/gx8/extras/vlad/miniconda/envs/pvac/lib/python3.6/site-packages/lib/pipeline.py", line 296, in top_result_filter
    TopScoreFilter(self.coverage_filter_out_path(), self.top_result_filter_out_path(), self.top_score_metric).execute()
  File "/g/data3/gx8/extras/vlad/miniconda/envs/pvac/lib/python3.6/site-packages/lib/top_score_filter.py", line 23, in execute
    amino_acid_change_position = line['Fusion Position']
KeyError: 'Fusion Position'
```

**Input and Output File**
[mysample_pvacfuse.zip](https://github.com/griffithlab/pVACtools/files/2385548/mysample_pvacfuse.zip)

**Comment by susannasiebert** [comment]
Thank you for this bug report. A pull request to fix this issues has been made (#196) and will be released with the next hotfix release.

**Comment by vladsaveliev** [comment]
Thank you so much for resolving it quickly!

**Comment by susannasiebert** [comment]
The PR has been merged and I made a new pVACtools release (1.0.9) that includes this bugfix. I'm resolving this issue but feel free to reopen if this issue continues to persist.
